plugins/org.eclipse.papyrus.uml.interaction.model/src/org/eclipse/papyrus/uml/interaction/model/spi/ElementRemovalCommandImpl.java

```diff
@@ -55,7 +55,7 @@ public RemovalCommandImpl(EditingDomain domain, Command command, EObject object)
 	/**
 	 * Creates a removal command which combines the result of the given removal commands.
 	 */
-	public RemovalCommandImpl(EditingDomain domain, List<RemovalCommand> commands) {
+	public ElementRemovalCommandImpl(EditingDomain domain, List<RemovalCommand<Element>> commands) {
```

Comment:
The parameter type should probably be `List<? extends RemovalCommand<? extends Element>>`.